_modules/hubble.py

```diff
@@ -218,6 +218,11 @@ def audit(configs=None,
     if not called_from_top and not results:
         results['Messages'] = 'No audits matched this host in the specified profiles.'
 
+    for error in ret.get('Error', []):
+      if not terse_results.has_key('Error'):
```

Comment:
Oops, this is because I was testing on another machine and (badly) copied my changes across! I originally only had this working for non-verbose, until I realised and moved it down the function and changed it to `results`.